milli/src/update/index_documents/mod.rs

```diff
@@ -322,6 +330,7 @@ where
 
         // Run extraction pipeline in parallel.
         pool.install(|| {
+            puffin::profile_scope!("split_grenad_by_chunks");
```

Comment:
This profile scope doesn't seem to only measure the `split_grenad_by_chunks` process but`data_from_obkv_documents` too.